### PR TITLE
Added "Whitelines" colour scheme

### PR DESCRIPTION
**Pull request by TEParsons**
Added a colour scheme styled after Whitelines notebooks, with a light grey background and guides drawn on in white.

### CODE REVIEW

**Comment by FichteFoll** [comment]
The package defines a `.sublime-color-scheme` and can thus only be used from build 3149 onwards. Please adjust that in the ST version selector.

You should definitely include screenshots in the readme of a color scheme.

**Comment by FichteFoll** [comment]
@TEParsons ping

File: repository/w.json [hunk 1]
```diff
@@ -370,11 +370,22 @@
 			"previous_names": ["White Night Theme"],
 			"releases": [
 				{
-					"sublime_text": "*",
+					"sublime_text": ">=3149",
```
Comment (rchl) [inline repository/w.json:373]: You've modifier wrong line.

File: repository/w.json [hunk 2]
```diff
@@ -370,11 +370,22 @@
...
 			]
 		},
+		{
+			"name": "Whitelines",
+			"details": "https://github.com/TEParsons/Whitelines",
+			"labels": ["colour scheme", "color scheme", "stationery", "light", "orange", "grey", "silver"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": "Whitelines"
```
Comment (rchl) [inline repository/w.json:385]: I think you want to set `tags` to `true`. Custom tag names only really make sense when you have two separate releases for different ST version.

What you should do to fix the bot issue is to create a tagged release on github.

**Comment by rchl** [comment]
The image links are broken in your readme. Use paths like `./_media/Foo.png`

File: repository/w.json (hunk 1 above)
Comment (TEParsons) [reply repository/w.json:373]: ...it's been a long day :')

File: repository/w.json (hunk 2 above)
Comment (TEParsons) [reply repository/w.json]: Thanks, I think this should do it?

**Comment by rchl** [comment]
I think it should be fine now. You can close and reopen PR to trigger the bot.

**Comment by FichteFoll** [comment]
You need to create a new version tag for your changes so that PC can pick them up.

**Pull request merged**